tests/ha/firewall_disable.pm

```diff
@@ -14,14 +14,26 @@ use base 'opensusebasetest';
 use strict;
 use testapi;
 use hacluster;
+use version_utils 'is_sle';
+
+sub stop_and_disable_firewall {
+    my $firewall = shift;
+
```

Comment:
Why not just access the member variable?

Reply:
Like this: `my ($firewall) = @_;`?